doc/nrf/templates/sample_README.rst

```diff
@@ -99,35 +217,57 @@ Troubleshooting*
 
 If the LEDs do not start blinking, check if the music is loud enough.
 
+
+Known issues and limitations*
+*****************************
+
+The sample only works with good music.
+
+References*
```

Comment:
I'm a bit concerned that these references might have more sense within overview or some other section before. Should we then duplicate the reference here?

Reply:
No, I don't think we should duplicate. But we have a few samples that have additional references (mostly to specs).